compiler/src/parsing/asttypes.re

```diff
@@ -115,9 +115,3 @@ type attribute = (loc(string), list(loc(string)));
 
 [@deriving (sexp, yojson)]
 type attributes = list(attribute);
-
-[@deriving (sexp, yojson)]
-type argument_label =
-  | Unlabeled
-  | Labeled(loc(string))
-  | Default(loc(string));
```

Comment:
Why was this moved out of Asttypes instead of being aliased in `parsetree.re` like we do for `mut_flag`, `rec_flag`, etc?

Reply:
I don't actually remember why I did this. I'll revert this back into asttypes

Reply:
Thanks, the diff looks great now!